recipes/python-omero/meta.yaml

```diff
@@ -25,9 +25,6 @@ requirements:
     - zeroc-ice =3.6.5
       
 test:
-  commands:
-    - omero --help
-    - omero version
```

Comment:
These give `FATAL: Running $PREFIX/bin/omero as root can corrupt your directory permissions.` but testing the command locally (without root) works fine.

Reply:
will solve by grepping for the error message